### DISCUSSION TITLE
How to convert data type in dataframe?

### DISCUSSION THREAD

**Discussion by dlintw** [nushell/nushell#11378] [Q&A] [answered]
df is a dataframe which type is str. I want to convert them into int, but fail.

```
> $df | dfr dtypes
#   column   dtype
 0   size     str
> $df
      #   size
      0   164
      1   164
    ...   ...
 138488   56
 138489   56

> $df | describe
dataframe
> $df | into int
only int, float, filesize, date, string, binary, duration, or bool input data is supported
```

**Comment by fdncred** [comment, marked as answer]
This is probably better asked in our #dataframe channel on discord where people who use dataframes lurk. My only guess would be to do `dfr into-nu` and then do `into int`.

**Comment by dlintw** [comment]
` (($df.size | dfr into-nu).size | into int) | dfr into-df`
